bylaws/009-funding.md

```diff
@@ -0,0 +1,47 @@
+# Funding
+
+## Scope and objectives
+
+PHP-FIG is, as it often happens in Open Source communities, an organization composed by volunteers only; this doesn't exclude the fact that the organization itself may have some costs to sustain itself, like for maintaining a couple of tools such as a site and an email account. For such and similar reasons, we need a way to raise and govern money inside our organization.
+
+This document defines the roles, rules and processes that must be followed inside the PHP-FIG in all matters regarding money, to guarantee fairness, transparency and impartiality. This document intentionally describes the only allowed processes and ways to raise, administer and spend money, so that any change to such processes or any variation in budget allocation requires a bylaw change vote. 
+
+## Fiscal hosting
+For handling money and fiscal matters, the PHP-FIG will use [OpenCollective](https://opencollective.com/) as a fiscal host.
+As for any other tool pertaining to the PHP-FIG, secretary will be the only ones to have full administrative access to PHP-FIG's OpenCollective account, but they will not have any decision power over it, and they are instead bounded by this document.
+
+All monetary contributions will flow through it, so that any tax matter will be handled automatically, and every financial decision will be published and publicly available through their budgetary tools.
```

Comment:
```suggestion
All monetary contributions will flow through OpenCollective, so that any tax matter will be handled automatically, and every financial decision will be published and publicly available through OpenCollective's budgetary tools.
```